src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx

```diff
@@ -91,7 +91,9 @@ export class ArtworkSidebarCommercialContainer extends React.Component<
   renderSaleMessage(saleMessage: string) {
     return (
       <Text variant="lg-display" data-test="SaleMessage">
-        {saleMessage === "Contact For Price" ? "Price on request" : saleMessage}
+        {saleMessage.toLowerCase() === "contact for price"
+          ? "Price on request"
+          : saleMessage}
```

Comment:
**suggestion**: maybe put this check into helper method? something like this:

```javascript
const checkIsContactForPrice = (message: string) => message.toLowerCase() === "contact for price"
```

Reply:
We're going to delete these files either way after the new Artwork screen is released so I was thinking of leaving that as it is.
This change is mainly for the transitional phase until we release it